client/src/components/Sidebar/common/Icon.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+export interface IconProps {
+  name: string;
+  className?: string;
+  title?: string;
+}
+
+/**
+ * Provide a `title` as an accessible label intended for screen readers.
+ */
+const Icon: React.FunctionComponent<IconProps> = ({ name, className, title }) => (
```

Comment:
This and `Button` are being duplicated from the rest of Wagtail, right? Is there a particular motivation for this?

Reply:
These are merged in a following PR: https://github.com/wagtail/wagtail/pull/7197